gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -94,8 +95,8 @@ func gatherOptions() options {
 
 	fs.IntVar(&o.flPort, "p", 8080, "port number on which to listen")
 
-	fs.StringVar(&o.flIcons, "i", "/icons", "path to the icons directory")
-	fs.StringVar(&o.flStyles, "s", "/styles", "path to the styles directory")
+	fs.StringVar(&o.flIcons, "i", getStaticFilePath("/icons"), "path to the icons directory")
```

Comment:
This is the reason why `embed` might not work

Reply:
Is anyone even using them? I think they only exist be because embedd didn't exist. We might let them default to an empty string and use an `embed.Fs` if their value is an empty string and read from the real fs is they are set.

Reply:
tbh I would be very surprised if anyone uses them. But if there is a solution to avoid backward incompatibility I would go for it. So yeah this is a good plan